src/components/atoms/Price/PriceUnit.tsx

```diff
@@ -48,7 +49,8 @@ export default function PriceUnit({
         <>
           <div>
             {Number.isNaN(Number(price)) ? '-' : formatPrice(price, locale)}{' '}
-            <span className={styles.symbol}>{symbol || 'OCEAN'}</span>
+            {/* do we want a default symbol? */}
```

Comment:
think it's fine without a default or fallback. We could define some base token in the app config but just querying the symbol seems more reliable anyway.